### ISSUE TITLE
I am in the 'There should be a button for deleting todos' part of V8 and my code looks just as it should, but will not respond to any input for deleting todos. What might be going on?

### ISSUE THREAD

**Issue by cranjis-mcbasketball** [gordonmzhu/questions#264]
### What is the associated Watch and Code lesson? 

https://watchandcode.com/courses/60264/lectures/1027560

### Include a live demo link so that I can run your code. 

https://plnkr.co/edit/n8YE5Iga7hV9lzllVA9l

### Explain the exact steps I need to take to replicate the problem. 

I am having trouble figuring out what you would have to do to replicate the problem because my code looks as it should according the video, and I have had no problems with the rest of the version 8 lessons. By the time I got to the ‘There should be a button for deleting todos’ lesson of Version 8 I felt like I was getting the hang of the Version so I attempted to write the code before watching the video and I learned (to my initial joy) that the code was completely correct after watching the video so then I ran the code. 

### What did you expect to happen? 

I expected my Delete Todo button to 1)delete the todo at the position specified in the input box and 2)clear the input field after the todo was deleted.  This is what the video described to me as to what should happen. My other buttons work properly. 

### What actually happened? 

After following the usual protocol by creating a Todo List, I attempted to delete the Todo at position ‘0’ but this did not happen. I understand that the new button should both delete the selected Todo and then the input field should go blank but neither of those things happened.

### Describe what you've done so far to troubleshoot the problem. 

I tried putting the debugger into the 'deleteTodo' method of the handlers object, but it seems as if the button is completely nonfunctional so nothing happened in my console. I then tried to add an additional debugger to the 'changeTodo' method of the handlers object which worked as expected but I was still never able to access the debugger in the deleteTodo method even when trying to 'step down' into the next function

**Comment by gordonmzhu** [comment]
Can you list out the possible things that might be going wrong?

**Comment by cranjis-mcbasketball** [comment]
Things I have tried based on what I *thought *might be going wrong:

1. What I thought: perhaps there was an inconsistent spelling (ie: position
vs positon)
What I did: I copy/pasted the entire code in the js section into the
console and no errors popped up so I imagine there were no slight typos in
the code as far as defining variables and/or keeping spellings consistent.

2. What I thought: A setting I had in the 'inspect' section of chrome was
causing it. What I did: I tried using safari instead which didn't work,
suggesting the problem is within something I have in Plunker.

3. What I thought: The deleteTodo method of the original todoList object
was not correct
What I did: I tested it out on the console with plunker preview target and
found that it worked just fine. I also checked through the lessons to make
sure I didn't alter it from the one we learned and I did not.

Unfortunately I am still troubleshooting but not having much luck with
pinpointing the issue. I suspect it is related to the HTML portion and/or
the user interface but I do not know what is wrong but am continuing to
look.
